Automattic/wc-calypso-bridge
# Update check to see if feature exists before adding

Fixes #747 

The previous check was checking for an array key, but as far as I can tell, WCA features are and have been using an array of feature keys not indexed by the key itself.

### Before
![Screen Shot 2021-12-03 at 2 57 47 PM](https://user-images.githubusercontent.com/10561050/144666533-b82812f1-6df2-41f4-9db7-1cedbb4fe424.png)


### After
![Screen Shot 2021-12-03 at 3 08 29 PM](https://user-images.githubusercontent.com/10561050/144666547-7f49cca0-f045-4b34-a355-2e859b672adf.png)

### Testing

1. Set up this repo to mimic the bridge dev mode and ecomm plans - https://github.com/Automattic/wc-calypso-bridge
2. Install and activate the latest `main` from WCA
3. Navigate to the WC Settings->Advanced->Features.
4. Make sure only one toggle is shown for the navigation
5. Optionally, install a really old version of WCA and/or WooCommerce to make sure a single toggle is still shown.
